app/Resources/views/Exception/error403.html.twig

```diff
@@ -0,0 +1,16 @@
+{% extends 'includes/base.html.twig' %}
+
+{% block body %}
+    <div class="container">
+        <h1>You are not allowed to do this! ERROR 403</h1>
+
+        {% if is_granted('IS_AUTHENTICATED_FULLY') %}
+            <h2>Ask the admin why you are not allowed to do that</h2>
```

Comment:
Meint IS_AUTHENTICATED_FULLY, dass der Benutzer eingeloggt ist?
Vielleicht schon ein Hinweis auf #38

Reply:
Genau das ist gemeint :) 

Wollen wir den Hinweis in einem neues Branch hinzufügen?

Reply:
So hab das jetzt schon hinzugefügt